framework/graphs/petsc_graph_partitioner.cc

```diff
@@ -93,7 +93,7 @@ PETScGraphPartitioner::Partition(const std::vector<std::vector<uint64_t>>& graph
     log.Log0Verbose1() << "Done copying to raw indices.";
 
     // Create adjacency matrix
-    Mat Adj;                         // Adjacency matrix
+    Mat Adj = nullptr;                         // Adjacency matrix
```

Comment:
You could probably remove the comment if you want...

Reply:
Fixed!